pombola/kenya/templates/facebook-experiment-politics.html

```diff
@@ -129,7 +129,7 @@
     </div>
 </div>
 
-<div id="ctas">
+<div id="ctas" hidden>
```

Comment:
IE earlier than 11 doesn't support the `hidden` attribute according to http://caniuse.com/#feat=hidden so maybe the CSS equivalent `display: none` would be better.